libsawtooth/src/store/mod.rs

```diff
@@ -23,6 +23,8 @@ pub mod receipt_store;
 pub mod redis;
 
 use std::convert::TryInto;
+pub use std::ops::Bound;
```

Comment:
You shouldn't have to re-export this, since it's a std library item.

Reply:
Fixed